docs/developer/soffits/README.md

```diff
@@ -0,0 +1,46 @@
+# uPortal Soffits
```

Comment:
Still [hoping](https://github.com/drewwills/Soffit/issues/16) for a name other than "Soffit".